src/Common/src/TypeSystem/IL/Stubs/DelegateMarshallingMethodThunk.cs

```diff
@@ -15,19 +15,21 @@ namespace Internal.IL.Stubs
     internal class DelegateMarshallingMethodThunk : ILStubMethod
     {
         private readonly TypeDesc _owningType;
-        private readonly TypeDesc _delegateType;
-        private readonly string _name;
+        private readonly MetadataType _delegateType;
         private readonly InteropStateManager _interopStateManager;
         private readonly MethodDesc _invokeMethod;
+        public readonly bool IsOpenStaticDelegate;
```

Comment:
Nit: Not sure if this is documented anywhere, but we generally only do `public readonly` fields on structs. Classes mostly have properties.